spec/ParseUser.spec.js

```diff
@@ -1265,6 +1265,36 @@ describe('Parse.User testing', () => {
     done();
   });
 
+  fit('log in with Facebook and save signed up User with verifyUserEmails=true and preventLoginWithUnverifiedEmail=true', async done => {
+    const emailAdapter = {
+      sendPasswordResetEmail: () => Promise.resolve(),
+      sendMail: () => Promise.resolve(),
+    };
+    await reconfigureServer({
+      appName: 'unused',
+      verifyUserEmails: true,
+      preventLoginWithUnverifiedEmail: true,
+      emailAdapter: {
+        module: emailAdapter,
+      },
+      publicServerURL: 'http://localhost:8378/1',
```

Comment:
I'm not sure theis option (and appname) is needed for your test. Could you please again look through all the code and remove the lines that are not needed to replicate the issue?